### PR TITLE
feature:GetAllUpdateBlog

### PR DESCRIPTION
**Pull request by maureen147**
# Fixes Issue/Linear Ticket

> This could be an existing issue or a linear ticket
>
> - Github Issue Example: My PR Closes #{ISSUE}
> - Linear Ticket Example: Fixes ID-#{ISSUE}

## Changes proposed

> Talk about the things you did eg. files changes, dependencies installed e.t.c

## Check List (Check all the applicable boxes)

:rotating_light:Please review the [style guide for contributing](add link here) and [guidelines for contributing](add link here) to this repository.

- [ ] My code follows the code style of this project.
- [ ] This PR does not contain plagiarized content.
- [ ] The title and description of the PR is clear and explains the approach.
- [ ] I am making a pull request against the **main branch** (left side).
- [ ] My commit messages styles matches our requested structure.
- [ ] My code additions will fail neither code linting checks nor unit test.
- [ ] I am only making changes to files I was requested to.

### CODE REVIEW

File: package.json
Comment (incredible-phoenix246) [inline package.json:1]: first of use 

npm i -g yarn

not installing it in the project

File: package-lock.json
Comment (incredible-phoenix246) [inline package-lock.json:1]: delete this

File: src/data-source.ts
Comment (incredible-phoenix246) [inline src/data-source.ts:1]: comment it out after you are done

File: src/models/blog.ts
Comment (incredible-phoenix246) [inline src/models/blog.ts:1]: explain what happened here

**Review by incredible-phoenix246 — CHANGES_REQUESTED** [review]

File: src/data-source.ts
Comment (maureen147) [reply src/data-source.ts:1]: done

**Comment by maureen147** [comment]
i have resolved my conflict

File: src/models/blog.ts
Comment (maureen147) [reply src/models/blog.ts:1]: included my own code

**Pull request closed without merging**